test/unit/plugins/providers/hyperv/driver_test.rb

```diff
@@ -91,7 +91,7 @@ def generate_error(msg)
     describe "#set_vm_integration_services" do
       it "should map known integration services names automatically" do
         expect(subject).to receive(:execute) do |name, args|
-          expect(args[:Name]).to eq("Shutdown")
+          expect(args[:Id]).to eq("9F8233AC-BE49-4C79-8EE3-E7E1985B2077")
```

Comment:
Please use the constant hash to match within the `eq` instead of the raw value. This just makes it easier to understand when reading it (instead of just seeing a magic value) and will be automatically updated if the value ever has to be modified for some reason.